src/server/controllers/index.ts

```diff
@@ -27,7 +27,7 @@ export function setupApiRoutes(app: express.Application): void {
   app.get('/api/users/me', authMiddleware, userController.getMe);
   app.post('/api/users/register', validate(userController.registerValidation), userController.registerUser);
   app.put('/api/users/resetpassword', validate(userController.resetPasswordValidation), userController.resetPassword);
-  app.put('/api/users/changepassword', authMiddleware, validate(userController.changePasswordValidation), userController.changePassword);
+  app.put('/api/users/changepassword', validate(userController.changePasswordValidation), userController.changePassword);
```

Comment:
Uhm, yeah. Makes sense. So then I'm thinking of two options:

1. Have same endpoint and then we can't use the authMiddleware here, we'd have to check the auth header manually in userController. Not sure if that can handled easily by passport.

2. Have two endpoints. Leave changepassword as is and use it for 'Account Settings' and then have a '/api/users/resetpasswordsteptwo' endpoint changes the password based on userId and resetPasswordId.

I personally like the second option better.